vendors/twtg/models/neon-vb_v4/model.yaml

```diff
@@ -0,0 +1,81 @@
+# Commercial name of the model
+name: TWTG NEON Vibration Sensor v4
+# Functional description of the product. Maximum 500 characters.
+description: "The NEON Vibration Sensor provides insight on vibrations (acceleration and velocity) of industrial equipment. Triggers can be set, based on which the sensors can send alerts."
+# Logo of the device
+logo: twtg-vb.png
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+# id: as defined under /profiles.
+# lorawanDeviceProfileID: as defined by LoRa Alliance in https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf.
+# The lorawanDeviceProfileID consists of 8 upper-case hexadecimal characters, 4 characters for VendorID + 4 characters for VendorProfileID. Please consider using single (') or double (") quotation.
+deviceProfileIds:
+    - id: twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA
```

Comment:
We copied this profile naming from the template, but we don't know where the `a` stands for in twtg_RFGroup1_1.0.4**a**_RP2-1.0.3_classA.

Should we keep it or remove it?

Reply:
Yes, it's preferable to remove the "a", as it's supposed to replace the revision "revA" for 1.0.4.
Thank you.

Reply:
Also, it seems the link to the datasheet isn't working